File: JobSearchProject/Models/ResumeModels/Resume.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace JobSearchProject.Models.ResumeModels
{
    public class Resume
    {
        public int Id { get; set; }

        [Required]
        [Column(TypeName = "varchar(MAX)")]
        public string Description { get; set; }

        [Required]
        public int Age { get; set; }
        
        public Status Status { get; set; }

        public bool Top { get; set; }

        [Required]
        [Column(TypeName = "varchar(50)")]
        public string ContactName { get; set; }

        public byte? Photo { get; set; }

        public int LocationId { get; set; }

        public virtual Location Location { get; set; }

        public int SpecializationId { get; set; }

        public virtual Specialization Specialization { get; set; }

        public List<Experience> Experiences { get; set; }
    }
}

File: JobSearchProject/ClientApp/src/app/models/resume.ts
import { Specialization } from "./specialization";
import { Location } from "./location";
import { Status } from "./enum";
import { Experience } from "./experience";

export class Resume {
    public id: number;
    public description: string;
    public age: number;
    public status: Status;
    public top: boolean;
    public contactName: string;
    public photo: string;
    public specialization: Specialization;
    public location: Location;
    public experiences: Experience[];

    constructor(
    ) {
        this.id = null;
        this.description = null;
        this.age = null;
        this.status = null;
        this.top = false;
        this.contactName = null;
        this.photo = null;
        this.specialization = new Specialization();
        this.location = new Location();
        this.experiences = new Array<Experience>()
    }
}
